### DISCUSSION TITLE
Disable zellij’s tmux prefix?

### DISCUSSION THREAD

**Discussion by LandfallSituated** [zellij-org/zellij#3254] [Q&A] [answered]
Sometimes there is only tmux on the remote machine, but it conflicts with the local tmux prefix of zellij. How to completely disable zellij’s compatibility with tmux prefix?

**Comment by xl4624** [comment, marked as answer]
I think you can either:

1. Explicitly unbind ctrl-b in your config file
```
shared_except "tmux" "locked" {
//  bind "Ctrl b" { SwitchToMode "Tmux"; }
    unbind "Ctrl b"
}
```

2. Change from `keybinds` to `keybinds clear-defaults=true` and delete the tmux and ctrl-b bind sections
```
// keybinds {
keybinds clear-defaults=true {
//  default binds without tmux here...
}
```
You could also clear the defaults only for the tmux section. Here are the [docs](https://zellij.dev/documentation/keybindings-binding) for overriding keys.

**Reply by LandfallSituated** [reply]
thanks, bro, it works for me!
